build/common.sh

```diff
@@ -395,6 +395,23 @@ function kube::build::build_image() {
 # $3 is the value to set the --pull flag for docker build; true by default
 # $4 is the set of --build-args for docker.
 function kube::build::docker_build() {
+  "${DOCKER[@]}" buildx >/dev/null 2>&1
```

Comment:
hmm, i thought we added `kube::util::ensure-docker-buildx`can you please look at reusing that?

```
[dims@dims-a01 08:18] ~/go/src/k8s.io/kubernetes ⟩ rg kube::util::ensure-docker-buildx
test/images/image-util.sh
117:  kube::util::ensure-docker-buildx

build/lib/release.sh
219:  kube::util::ensure-docker-buildx

hack/lib/util.sh
710:# kube::util::ensure-docker-buildx
713:function kube::util::ensure-docker-buildx {
```

Reply:
yep, that is handy :) thanks updated
PTAL @dims